src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -1158,7 +1195,7 @@ def serialize(
 
         with sentry_sdk.start_span(op="building.trace", description="light trace"):
             # Check if the event is an orphan_error
-            if not snuba_event and not nodestore_event and allow_orphan_errors:
+            if not snuba_event or not nodestore_event:
```

Comment:
Why did this `and` change into an `or`?

Reply:
👀 uh. no idea good catch

Reply:
Oh nevermind, this is a typing catch, the rest of this function needs both snuba_event and nodestore_event to be defined, so if either isn't defined we want to do the following block